superset-frontend/src/dashboard/components/PublishedStatus/PublishedStatus.test.tsx

```diff
@@ -30,15 +30,17 @@ const defaultProps = {
 };
 
 test('renders with unpublished status and readonly permissions', async () => {
-  const tooltip = /This dashboard is not published which means it will not show up in the list of dashboards/;
+  const tooltip =
+    /This dashboard is not published which means it will not show up in the list of dashboards/;
```

Comment:
We should use `t()` here (and probably a million other plces)

Reply:
there is in unittest, this may not require i18n.

Reply:
Hah, I was just skimming more than reading... oops!